src/jit/lsra.cpp

```diff
@@ -4564,6 +4564,8 @@ void LinearScan::setFrameType()
         }
     }
 
+    frameType = FT_EBP_FRAME;
```

Comment:
This was added just to get tests working so I didn't bother too much with it. I think it should be in a different PR but maybe just a separate commit is fine (as long as the above feedback is addressed). Thoughts?